apps/daimo-mobile/src/App.tsx

```diff
@@ -34,10 +43,73 @@ export default function App() {
 }
 
 function AppBody() {
+  const bottomSheetRef = useRef<BottomSheet>(null);
+
+  const snapPoints = useMemo(() => ["50%"], []);
+
+  useEffect(() => {
+    const subscription = RNShake.addListener(() => {
+      bottomSheetRef.current?.expand();
+    });
+
+    return () => {
+      subscription.remove();
+    };
+  }, []);
+
+  const renderBackdrop = useCallback(
+    (props: BottomSheetDefaultBackdropProps) => (
+      <BottomSheetBackdrop
+        {...props}
+        disappearsOnIndex={-1}
+        appearsOnIndex={0}
+        pressBehavior="close"
+      />
+    ),
+    []
+  );
+
+  const noop = useCallback(() => {}, []);
+
   return (
     <SafeAreaProvider>
       <TabNav />
       <StatusBar style="auto" />
+      <BottomSheet
+        handleComponent={ScrollPellet}
+        backdropComponent={renderBackdrop}
+        ref={bottomSheetRef}
+        index={-1}
+        snapPoints={snapPoints}
+        enablePanDownToClose
+      >
+        <View style={styles.contentContainer}>
+          <View>
+            <TextH3>Did something go wrong?</TextH3>
+            <Spacer h={12} />
+            <TextLight>
+              Send us debug log to help us realise what went wrong
```

Comment:
i'll defer to @michelle-chiu for the copy

otherwise LGTM

Reply:
Design detail—can we cut down the spacing between the heading & top lip of the component as much as possible? Looks great otherwise :)